### ISSUE TITLE
Datatype

### ISSUE THREAD

**Issue by ahmetertem** [brianc/node-sql#363]
I checked few files to see is there any support for each dialect's datatype. For example there is no `datetime` datatype on sqlite3 (*yes i may do whatever what i want with sqlite3's datatypes ofc*) but MySQL has datetime or timestamp for example. Or `bit` datatype is different before `MySQL 5.6.X`  (*i may wrong on this, i do not remember well*) but after that version, bit assumed like `tinyint(1) default 0` etc...

**TL;DR**: I want to easily create tables with selected dialect without changing datatypes 🚀 

Can you support this as well ?
